src/main/java/org/broadinstitute/hellbender/utils/codecs/SVCallRecordCodec.java

```diff
@@ -0,0 +1,76 @@
+package org.broadinstitute.hellbender.utils.codecs;
+
+import com.google.common.base.Splitter;
+import htsjdk.tribble.AsciiFeatureCodec;
+import htsjdk.tribble.index.tabix.TabixFormat;
+import htsjdk.tribble.readers.LineIterator;
+import htsjdk.variant.variantcontext.GenotypeBuilder;
+import htsjdk.variant.variantcontext.StructuralVariantType;
+import org.broadinstitute.hellbender.tools.sv.SVCallRecord;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class SVCallRecordCodec extends AsciiFeatureCodec<SVCallRecord> {
+
+    public static final String FORMAT_SUFFIX = ".tsv.gz";
```

Comment:
This class will probably evolve later, but it would be best to change this one line now. Let's make the suffix `".sv_calls.tsv.gz"` otherwise we could create trouble with feature codec collisions.